integrations/tensorflow/e2e/exported_names_test.py

```diff
@@ -25,19 +25,23 @@ def exported_fn(self):
 
   # No input_signature, so it cannot be imported by the SavedModel importer.
   # We need to ensure that
-  @tf.function
-  def unreachable_fn(self, x):
-    return x
+  @tf.function(input_signature=[])
+  def unreachable_fn(self):
+    return tf.constant([24.])
 
 
 # To pass a set of exported names for the module, instead of passing just a
 # module ctor, instead pass a pair `(ctor, [list, of, exported, names])`.
-@tf_test_utils.compile_modules(
-    dont_export_everything=(DontExportEverything, ["exported_fn"]))
+@tf_test_utils.compile_module(
+    DontExportEverything, exported_names=["exported_fn"])
 class DontExportEverythingTest(tf_test_utils.SavedModelTestCase):
 
-  def test_dont_export_everything(self):
-    self.modules.dont_export_everything.all.exported_fn().assert_all_close()
+  def test_exported_name(self):
+    self.get_module().exported_fn().assert_all_close()
+
+  def test_unreachable_name(self):
+    with self.assertRaises(AttributeError):
```

Comment:
I guess, unreachable_fn is reachable in this test, so AttributeError will not be raised and test will fail

Reply:
It is not reachable by any of the `iree_*` backends, but appears to be reachable by the`tf` backends. I think this might make sense, so maybe we want to exclude `tf` from this test? We're not actually checking correctness anywhere.

Reply:
I excluded the tensorflow backend from this test since it's not actually being passed through `pyiree`s compiler.